spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/simple/SimpleExportConfiguration.java

```diff
@@ -37,10 +38,10 @@
 public class SimpleExportConfiguration {
 
 	@Bean
-	@ConditionalOnProperty(value = "spring.metrics.export.simple.enabled", matchIfMissing = true)
+	@ConditionalOnProperty(value = "spring.metrics.simple.enabled", matchIfMissing = true)
```

Comment:
Can you revert this line please? We'd like to keep all these properties beneath `spring.metrics.export`